src/metaschema/oscal_catalog_metaschema.xml

```diff
@@ -146,6 +146,7 @@
                   </allowed-values>
                   <allowed-values target="part[has-oscal-namespace('http://csrc.nist.gov/ns/oscal')]/@name">
                         <enum value="overview">An introduction to a control or a group of controls.</enum>
+                        <enum value="statement">A set of implementation requirements.</enum>
```

Comment:
As I mentioned in the issue #1949 adding statement here makes uses of statement in `control` more confusing and murky. I don't think making existing terms more confusing is desirable.

There is no sense in having requirements in a `group`. They do not participate in the OSCAL models the same way as a `control`, since only a `control` can be referenced in an SSP implementation statement or selected in a profile. If the intent is to have implementation requirements, then a nested hierarchy of controls is much more fitting in the OSCAL models. There is `@class` on `control` to identify different types "controls".

Reply:
@david-waltermire-nist - Thank you for your feedback. @Arminta-Jenkins-NIST's mote reflects your comment in the issue, and I am aware of that. 
 ISO 27001, 27002, 27017 and 800-53 are using slightly different terminologies for controls and requirements. ISO has requirements and controls, while in 800-53, controls which are sets of requirements. 
I know you want OSCAL to be flexible to represent as many regulatory frameworks as possible, without extensions that require registries, not just 800-53 and FedRAMP. CSF v2 is one other example. 
So, even if a CSF function is not encoded as a <part  name="statement" ..> as proposed,
1. how will you name the part for `GOVERN=Establish and monitor the organization's cybersecurity risk management strategy, expectations, and policy .` if it is not a `statement`? It is not `description,` `overview` or `definition`, since it is something actionable. What other English word will define it best? I can modify the PR and propose a CSF namespace with such constraint and include it in core OSCAL as we did with RMF. I was trying to avoid it and keep a clean core OSCAL, but if you think following the RMF support approach is better, then I'll do so.
2.  putting aside CSF for this bullet discussion, a `statement` being define as a "control implementation requirement" causes issues for others (ISO expressed it in plain WG1 meeting, ACSC did it too - suggesting it to be a recommendation).  So, if 800-53 controls have statements, where a `statement="a set or implementation requirements or recommendations" ` - a definition that would accommodate ISO, ACSC/ISM - do you see a problem? If yes, can you please explain how is it causing murkiness for FedRAMP? Maybe then FedRAMP that has already a namespace well recognized and documented extensions can tighten it for FedRAMP use?

Reply:
Excerpt from ISO 27001:2017 : "International Standard has been prepared to provide **requirements** for establishing, implementing, maintaining and continually improving an information security management system. "  [...]  "Organizations can design controls as required, or identify them from any source." [...] "NOTE 1 Annex A contains a comprehensive list of control objectives and controls. Users of this International Standard are directed to Annex A to ensure that no necessary controls are overlooked.'  ISO 27001 defines requirements that point to controls. The automation with OSCAL made it into 27017 too. We both worked to convince 27002 to provide the standard in OSCAL. I am only looking for OSCAL adoption and flexibility for as many users as possible.

Reply:
Just noting in passing that the values are open-ended precisely so organizations do not have to design one another's solutions. The question should not be about aligning all the myriad application profiles, but about allowing them to work without inhibition. Any alignment that happens after that is nice, as long as it is not a blocker.

In my view the discussion should focus on how to make the stipulated value constraints (if any adjustments are needed) both *most useful* for those that need them, and *least noisy* (ideally silent) for those that don't. Anything else is up to applications and discussions such as those both raised are out of scope for the core spec (while they are in scope for their respective specs).

Would a requirement for 'less noise' be satisfied by an `allow-other='yes'` here?

Would it help the general case if no constraint were asserted here and this was left to FedRAMP or other users/consumers? The current constraint represents the SP800-53 case only, so far.

Reply:
value="statement" was replaced with value="instruction".